src/translations/index.ts

```diff
@@ -55,6 +55,13 @@ import ViVn_TRANSLATIONS from './vi-VN.json';
 import ZhCn_TRANSLATIONS from './zh-CN.json';
 import ZhHk_TRANSLATIONS from './zh-HK.json';
 import ZuZa_TRANSLATIONS from './zu-ZA.json';
+import DeAt_TRANSLATIONS from './de-AT.json';
+import DeCh_TRANSLATIONS from './de-CH.json';
+import ItCh_TRANSLATIONS from './it-CH.json';
+import FrCh_TRANSLATIONS from './fr-CH.json';
+import NlBe_TRANSLATIONS from './nl-BE.json';
+import HeIl_TRANSLATIONS from './he-IL.json';
+import EnNz_TRANSLATIONS from './en-NZ.json';
```

Comment:
FYI these are not displayed as options unless set in `selectableLanguages` below

Reply:
yup no problem

Reply:
I'll add Hebrew since it's a totally new language

Reply:
cool